docs/migration.md

```diff
@@ -1,235 +1,288 @@
+| calculate | `rdiff-backup --calculate-average {stat-file1} {state-files...}` | `rdiff-backup calculate [--method average] {stat-file1} {state-files...}` |
+| compare by metadata with the latest increment | `rdiff-backup --compare {source-dir} {target-dir}` | `rdiff-backup compare [--at now] [--method meta] {source-dir} {target-dir}` |
+| compare by metadata at given date/time | `rdiff-backup --compare-at-time {datetime} {source-dir} {target-dir}` | `rdiff-backup compare --at {datetime} [--method meta] {source-dir} {target-dir}` |
+| full compare with the latest increment | `rdiff-backup --compare-full {source-dir} {target-dir}` | `rdiff-backup compare [--at now] --method full {source-dir} {target-dir}` |
+| full compare at given date/time | `rdiff-backup --compare-full-at-time {datetime} {source-dir} {target-dir}` | `rdiff-backup compare --at {datetime} --method full {source-dir} {target-dir}` |
+| compare by hash with the latest increment | `rdiff-backup --compare-hash {source-dir} {target-dir}` | `rdiff-backup compare [--at now] --method hash {source-dir} {target-dir}` |
+| compare by hash at given date/time | `rdiff-backup --compare-hash-at-time {datetime} {source-dir} {target-dir}` | `rdiff-backup compare --at {datetime} --method hash {source-dir} {target-dir}` |
+| list files at given time in backup repo | `rdiff-backup --list-at-time {datetime} {backup-repo}` | `rdiff-backup list files --at {datetime} {backup-repo}` |
+| list files changed since given time in backup repo | `rdiff-backup --list-changed-since {datetime} {backup-repo}` | `rdiff-backup list files --changed-since {datetime} {backup-repo}` |
+| list increments in backup repo | `rdiff-backup --list-increments {backup-repo}` | `rdiff-backup list increments {backup-repo}` |
+| list increment sizes in backup repo | `rdiff-backup --list-increment-sizes {backup-repo}` | `rdiff-backup list increments --size {backup-repo}` |
+| check and correct a backup repo in case of failed backup | `rdiff-backup --check-destination-dir {backup-repo}` | `rdiff-backup regress {backup-repo}` |
+| Restore from a backup repo | `rdiff-backup -r {datetime} {backup-repo} {target-dir}` (`-r` or `--restore-as-of`) | `rdiff-backup restore --at {datetime} {backup-repo} {target-dir}` |
+| Restore an increment file | `rdiff-backup {backup-repo}/rdiff-backup-data/{dated-increment} {target-dir}` | `rdiff-backup restore [--increment] {backup-repo}/rdiff-backup-data/{dated-increment} {target-dir}` |
+| Start a server | `rdiff-backup --server` | `rdiff-backup server` |
+| Test one or more server connections | `rdiff-backup --test-server {remote-locations...}` | `rdiff-backup test {remote-locations...}` |
+| Verify hashes of last backup | `rdiff-backup --verify {backup-repo}` | `rdiff-backup verify [--at now] {backup-repo}` |
+| Verify hashes of backup at given time | `rdiff-backup --verify-at-time {datetime} {backup-repo}` | `rdiff-backup verify --at {datetime} {backup-repo}` |
```

Comment:
It certainly an improvement for usability.

Did you manage to generate a man page for the new CLI ?

Reply:
Yes, if you have installed pandoc, it should work flawlessly, just call e.g. `pip install .` and the new man page should be installed, whereas the old one is installed as rdiff-backup-old.